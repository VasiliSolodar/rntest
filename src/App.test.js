import React from "react";
import {shallow} from "enzyme";
// Note: test renderer must be required after react-native.
// import renderer from "react-test-renderer";
import App from "./App";

describe("App", () => {
  it("matches snapshot", () => {
    expect(getComponent()).toMatchSnapshot();
  });

  const getComponent = props => {
    const parsedProps = {
      ...props,
    };

    return shallow(<App {...parsedProps} />);
  };
});
