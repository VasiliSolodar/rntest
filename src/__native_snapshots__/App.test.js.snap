// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <AnimatedComponent
    source={
      Object {
        "testUri": "../../../src/logo.png",
      }
    }
    style={
      Array [
        Object {
          "height": 300,
          "width": 300,
        },
        Object {
          "transform": Array [
            Object {
              "rotate": "0deg",
            },
          ],
        },
      ]
    }
  />
  <Text
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    Create React Native Web App
  </Text>
  <Text>
    Open up src/App.js to start working on your app!
  </Text>
  <Text>
    Changes you make will automatically reload.
  </Text>
  <Text>
    Shake your phone to open the developer menu.
  </Text>
  <TouchableHighlight
    activeOpacity={0.85}
    delayPressOut={100}
    onPress={[Function]}
    style={
      Object {
        "backgroundColor": "#1B95E0",
        "borderRadius": 3,
        "marginTop": 10,
        "marginVertical": 10,
        "padding": 20,
      }
    }
    underlayColor="#0A84D0"
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    >
      Rotate Logo
    </Text>
  </TouchableHighlight>
</View>
`;
